src/blog/2024/04/beginers-guide-to-node-red-varriables.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Beginner's Guide to Node-RED Variables
+subtitle: Understanding Flow, Global, and Environment Variables in Node-RED.
+description: Understand Node-RED node, flow, global, and environment variables. Learn to initiate, retrieve, and manage them effortlessly.
+date: 2024-04-08
+authors: ["sumit-shinde"]
+image:
+tags:
+   - posts
+   - node-red
+   - variables
+   - global
+   - context
```

Comment:
this is a very broad keywword. Think about how it will be searched and the intent of the search.  Does this article satisfy that intent?

Reply:
Yes, I am sure that the updates we made to the keyword will satisfy the search intent. However, I'm unsure if adding such broad keywords would be appropriate. wdyt?